backend/.env.local

```diff
@@ -1,3 +1,4 @@
-RECORD_TYPES = ["Wikipedia"]
+RECORD_TYPES = ["wikipedia"]
 OUTPUT_FILE_NAMES = ["mini-wikipedia.output.txt"]
 ANTI_RECOMMENDER_TYPE = "OpenAI"
+CI = false
```

Comment:
Convention is that a "boolean" environment variable like CI is not set rather than being set to a false value. In GitHub Actions, if "CI" is set to anything then it's true, otherwise it's false.

Reply:
Most CI environments use that convention.